tasks/tasks/transformation_processors.py

```diff
@@ -131,7 +131,7 @@ def clip_time(context, filename=None, **kwargs):
 
     response = {
         "message": "Time values not clipped, some information was not provided (time column or time range list).",
-        "dataframe": json.loads(json.dumps(original_dataframe)),
+        "dataframe": original_dataframe.to_json(),
```

Comment:
Switching to this smoothly handled all the missing arguments errors instead of throwing backend errors. Seems to work and be the built in dataframe function, not sure if there are any drawbacks?